File: pkg/src/com/jackmeng/prismix/ux/intf/Intf.java
package com.jackmeng.prismix.ux.intf;

import com.jackmeng.stl.stl_Callback;

import java.util.function.Consumer;

public final class Intf
{
	private Intf() {}

	public static final Runnable DEAD_VOID_VOID = () -> {};

	public static final Consumer<?> DEAD_T_VOID = e -> {};

	public static final stl_Callback<?, ?> DEAD_T_T = e -> null;
}

File: pkg/src/com/jackmeng/prismix/ux/intf/Intf.kt
package com.jackmeng.prismix.ux.intf

import com.jackmeng.stl.stl_Callback
import java.util.function.Consumer

object Intf
{
	val DEAD_VOID_VOID=Runnable {}
	val DEAD_T_VOID:Consumer<*> =Consumer { e:Any?-> }
	val DEAD_T_T:stl_Callback<* , *> =stl_Callback<Any? , Any> { e:Any?-> null }
}